packages/extension/src/ui/features/accounts/AccountScreenEmpty.tsx

```diff
@@ -44,6 +44,7 @@ export const AccountScreenEmpty: FC<AccountScreenEmptyProps> = ({
       const result = await discoverAccounts(currentNetwork.id)
       if (result === "error") {
```

Comment:
@h0ngcha0 Do we support manual discovery since this automatic discovery could fail?

Reply:
Yes, there is a `Discover Accounts` button in the developer settings.